crates/wasi/src/p3/clocks/mod.rs

```diff
@@ -0,0 +1,71 @@
+mod host;
+
+use crate::clocks::{WasiClocksImpl, WasiClocksView};
+use crate::p3::bindings::clocks;
+use wasmtime::component::{HasData, Linker};
+
+/// Add all WASI interfaces from this module into the `linker` provided.
```

Comment:
Could this documentation, and I imagine all other related documentation too for similar functions, also have a clickable link to the "top-level" `add_to_linker` in case folks land on these docs before seeing that?

Reply:
<img width="989" height="391" alt="Screenshot 2025-07-16 at 12 43 14" src="https://github.com/user-attachments/assets/c6346e85-a43d-4ed3-905c-0941033096e9" />